oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/CurrentLocationInformation/CurrentLocationInformation.jsx

```diff
@@ -46,17 +47,17 @@ const NewChildItemMessage = ({ newChildItemParentId }) => {
       {!isLoading && data && (
         <VocabularyBreadcrumbMessage
           header={i18next.t("newChildItemMessage", {
-            item: data?.title?.cs,
+            item: data.title[i18next.language],
```

Comment:
Lets use I18nString from oarepo-ui for this :)

Reply:
@mirekys Ahoj, I tried doing this at first, but as it is react component and it is being interpolated into the translation, it does not seem to work. I have looked into I18next react and they do offer some ways how to translate some react components. I have tried using this, but I cannot get it to work somehow....